app/os/main.cpp

```diff
@@ -333,28 +333,20 @@ namespace robot {
 
 	}	// namespace internal
 
-	auto controller = RobotController {
-		internal::sleep_timeout,
-		battery::cells,
-		internal::serialnumberkit,
-		firmware::kit,
-		motors::left::motor,
-		motors::right::motor,
-		leds::ears,
-		leds::belt,
-		leds::kit,
-		display::internal::corelcd,
-		display::videokit,
-		behaviorkit,
-		commandkit,
-	};
-
-	void emergencyStop(const MagicCard &card)
-	{
-		if (card == MagicCard::emergency_stop) {
-			controller.raiseEmergencyStop();
-		}
-	}
+	auto controller = RobotController {internal::sleep_timeout,
+									   battery::cells,
+									   internal::serialnumberkit,
+									   firmware::kit,
+									   motors::left::motor,
+									   motors::right::motor,
+									   leds::ears,
+									   leds::belt,
+									   leds::kit,
+									   display::internal::corelcd,
+									   display::videokit,
+									   behaviorkit,
+									   commandkit,
+									   rfidkit};
```

Comment:
remettre une virgule ici

```suggestion
									   rfidkit,};
```